### DISCUSSION TITLE
Theme not work

### DISCUSSION THREAD

**Discussion by teguhsatria892** [inocan-group/vue3-google-map#232] [Q&A] [answered]
I got warning about styles

Invalid prop: type check failed for prop "styles". Expected Array, got String with value "aubergine"

```
                        <GoogleMap 
                            ref="googleMapDetail" 
                            style="width: 100%; height: 50vh" 
                            :api-key="googleMapData.apiKey" 
                            :center="googleMapData.center" 
                            :zoom="googleMapData.zoom"
                            :styles="'aubergine'"/>
```

whats wrong about this code?

Sample code
https://codesandbox.io/s/vue3-google-map-demo-forked-6wrdh5?file=/src/App.vue

I using developer mode API google map

**Comment by HusamElbashir** [comment, marked as answer]
Hi @teguhsatria892 

Please make sure you're using the latest version. Themes should now be imported from `vue3-google-map/themes` and passed through the `styles` prop. Example:
https://codesandbox.io/s/vue3-google-map-demo-forked-7hriki?file=/src/App.vue

Note that you'll need an API key for themes to work correctly

**Comment by teguhsatria892** [comment]
thanks a lot, i will try
